language/stdlib/modules/SlidingNonce.move

```diff
@@ -72,6 +72,10 @@ module SlidingNonce {
         move_to(account, SlidingNonce {  min_nonce: 0, nonce_mask: 0 });
     }
 
+    public fun assert_holds_sliding_nonce(addr: address) {
```

Comment:
I think it would be better to define:
```
public fun has_nonce(addr: address): bool {
  exists<SlidingNonce>(addr)
}
```
? This was
- other modules can check existence without triggering an abort
- we won't need to define an error code that is only used in tests

Reply:
Done